ui/src/app/utils/access-utils.spec.tsx

```diff
@@ -227,6 +232,7 @@ describe('computeRenewalDisplayDates', () => {
       nextReviewDate: `${displayDateWithoutHours(expirationDate)} (${
         EXPIRATION_DAYS - completionDaysPast
       } days)`,
+      moduleStatus: AccessModulesStatus.COMPLETE,
```

Comment:
"expiring soon"

Reply:
i need to understand this, will contact you offline for this logic